### DISCUSSION TITLE
Unknown reason for missing-issue-template tag, not able to correct

### DISCUSSION THREAD

**Discussion by kofaysi** [bromite/bromite#1450] [Q&A] [answered]
Can anybody, please, tell me, what is wrong with the report to issue? https://github.com/bromite/bromite/issues/1448 I used the predefined template, I deleted no comment, I added a lot of information. Still, issue has not been accepted yet.

**Comment by csagan5** [comment, marked as answer]
`91.0.4472.158` is an old version and it is a bug which is reproducible with Chromium.

**Reply by kofaysi** [reply]
Thank you! So the template had been correct all the time. The information supplied was obsolete. Maybe a different label and bot message shall be used for this kind of check: inaccurate-specification. Should be mentioned in comments of the template next to the related help questions:

Issues not related to the last published build are closed automatically

Issues reproducible with the Chromium are closed automatically

(Although I do not understand, why should be issues reproducible on Chromium ignored.)

**Reply by csagan5** [reply]
> (Although I do not understand, why should be issues reproducible on Chromium ignored.)

It is explained in the issue template:

> Please pick the same version of Chromium as Bromite from here: https://github.com/bromite/chromium/releases
> If the bug is reproducible then it might be a configuration issue or an upstream bug. Upstream bugs can be reported on the [Chromium issue tracker](https://bugs.chromium.org/p/chromium/issues/list) and do not forget to read [Chromium project bug reporting guidelines](https://www.chromium.org/for-testers/bug-reporting-guidelines) first.
> 
> If the bug is related to functionality that does not exist in Chromium then answer "No".
